test/tools/jsontests/BlockChainTests.cpp

```diff
@@ -183,64 +179,48 @@ json_spirit::mValue doBlockchainTestNoLog(json_spirit::mValue const& _input, boo
 				dev::test::TestBlockChain::s_sealEngineNetwork = network;
 				string newtestname = testname + "_" + test::netIdToString(network);
 
-				json_spirit::mObject jObj = o;
-				if (o.count("expect"))
+				json_spirit::mObject jObjOutput = inputTest;
+				if (inputTest.count("expect"))
 				{
 					//prepare the corresponding expect section for the test
```

Comment:
need to add a comment here that from one test with general expect section we write  'forknumber' tests with individual expect section to the output test which will again be used as a filler and overwriten.